src/compiler/diagnosticMessages.json

```diff
@@ -751,8 +751,22 @@
         "category": "Error",
         "code": 1235
     },
-
-
+    "'{0}' modifier can only appear on a class or member function declaration.": {

```

Comment:
Changed in the diagnostic messages and my comments, but I left it otherwise unchanged (for another PR...).